docs/source/developers/cpp/building.rst

```diff
@@ -60,6 +60,17 @@ On Alpine Linux:
            g++ \
            gcc \
            make
+           
+ On Fedora Linux:
+
+.. code-block:: shell
+
+   sudo dnf install \
+        gcc \
+        gcc-c++ \
+        make \
+        automake \
+        cmake      
```

Comment:
Could you sort this list in alphabetical order like others?

Reply:
Done